kalite/packages/bundled/securesync/management/commands/register.py

```diff
@@ -61,7 +61,7 @@ def handle(self, *args, **options):
         csrftoken = s.cookies['csrftoken_central']
         login_data = dict(username=options["username"], password=options["password"], csrfmiddlewaretoken=csrftoken, next='/')
         r = s.post(login_url, data=login_data, headers={"Referer": login_url})
-        assert r.status_code == 200, "Error logging into central server: " + r.content
+        assert r.status_code == 200 and "Incorrect user" not in r.content, "Error logging into central server: " + r.content
```

Comment:
The change looks like a desired precision from the commit msg: 34a27be96b2feb8bc09582ef401ced3cb2959ddb